src/modules/launcher/Microsoft.Launcher/dllmain.cpp

```diff
@@ -132,6 +132,11 @@ class Microsoft_Launcher : public PowertoyModuleIface {
    // Enable the powertoy
   virtual void enable()
   {
+      if (Is19H1OrHigher() == false)
```

Comment:
Let's uniform if's with function calls :) This one has explicit comparison, while the one in `settings_window.cpp ` doesn't.
Proposal: Change this one to `if (!Is19H1OrHigher())`

Reply:
We will revisit the code in 0.19 since it's currently duplicating existing code and we need to refactor it.